engine/apps/zvonok/zvonok_phone_provider.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import requests
+import os
+
+from random import randint
+
+from django.core.cache import cache
+
+from apps.base.models import LiveSetting
+from apps.base.utils import live_settings
+
+from apps.phone_notifications.exceptions import (
+    FailedToFinishVerification,
+    FailedToMakeCall,
+    FailedToSendSMS,
+    FailedToStartVerification,
+)
+from apps.phone_notifications.phone_provider import PhoneProvider, ProviderFlags
+
+ZVONOK_CALL_URL = 'https://zvonok.com/manager/cabapi_external/api/v1/phones/call/'
+ZVONOK_TIMEOUT = os.environ.get("ZVONOK_TIMEOUT", default=30)
```

Comment:
let's set ZVONOK_TIMEOUT to 10 seconds for now, without ability to configure it from env, I believe it's unnecessary